test-app/ios-uikit/TestApp/ButtonBinding.swift

```diff
@@ -48,7 +48,7 @@ class ButtonBinding: Button {
     
     @objc func clicked() {
         if (self.onClick != nil) {
-            self.onClick()
+            self.onClick?()
```

Comment:
I think `self.onClick?()` without `if` is better 😂

Reply:
This has nothing to do with the problem itself. Prioritize solving the actual problem. The main reason is that the calling method is wrong. Adding if is more standard.

Reply:
But why would we execute the `if` when adding the conditional chaining operator also performs a conditional? Aren't the two equivalent thus redundant?